src/Navigation/StackNavigation.js

```diff
@@ -127,12 +138,14 @@ function DrawerRoutes() {
 
 export default function StackNavigation() {
   return (
-    <stackNavig.Navigator initialRouteName="Log In">
+    <stackNavig.Navigator initialRouteName="Terms">
+      {/* CHANGE BACK TO LOG IN , remove terms */}
       <stackNavig.Screen name="Log In" component={LoginScreen} options={{ headerShown: false }} />
       <stackNavig.Screen name="Sign Up" component={SignUpScreen} options={{ headerShown: false }} />
       <stackNavig.Screen name="Sign Up Confirmation" component={SignUpConfirmation} options={{ headerShown: false }} />
       <stackNavig.Screen name="Marketplace" component={DrawerRoutes} options={{ headerShown: false }} />
       <stackNavig.Screen name="Profile" component={ProfileScreen} />
+      <stackNavig.Screen name="Terms" component={TermsScreen} options={{ headerShown: false }} />
```

Comment:
I think there may have been of overlap with the work I did since I did not put `Terms` on a separate screen. I consolidated it with the one `SignUpScreen` because a user cannot be created until they agree to the terms and conditions. I think we can merge this into master and then adjust later with my PR.